PF-build.sh

```diff
@@ -1303,7 +1303,7 @@ create_multi_firmware()
         echo "$(tput setaf 3)"
         ./lang-build.sh || failures 25
         # build community languages
-        ./lang-community.sh || failures 25
+        #./lang-community.sh || failures 25
```

Comment:
I assume this is just a temporary comment

Reply:
No, it is not needed anymore.

The updated lang scripts handle this.
`config.sh` is checking which community languages are active in `config.h` and sets the additional languages to be created and added:
```
config.sh started
 Arduino main folder: OK
 Arduino builder: OK
 AVR gcc tools:
   objcopy OK
   objdump OK
 Output folder: OK
 Objects folder: OK
 Generated elf file: OK
 Generated hex file: OK
 Languages: cz de es fr it pl
 Community languages: nl sv da sl hu lb hr 
config.sh finished with success
``` 
See https://github.com/prusa3d/Prusa-Firmware/pull/3336/files#diff-b94c0240e4353fc9197fdb3a55217a7092036fb62d84aa30c35dfe648b52a89dR39-R55